### PR TITLE
chore: add GitHub action for deploy to surge

### PR DESCRIPTION
**Pull request by vepor**
<br/><br/><br/><url>LiveURL: https://orbit-chore-surge-action.surge.sh</url>

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    runs-on: ubuntu-latest
+    name: Deploying to surge
+    steps:
+      - uses: actions/checkout@v2
+      - name: restore lerna
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
```
Comment (silvenon) [inline .github/workflows/deploy.yml:22]:
```suggestion
          path: **/node_modules
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      - name: Fire deployment
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Trying echo
+        run: echo "${GITHUB_REF#refs/heads/}"
+
+      - name: Install lerna
+        run: npm install -g lerna
```
Comment (silvenon) [inline .github/workflows/deploy.yml:34]: If this is for `lerna bootstrap`, Lerna is already a dependency of this project.

**Review by silvenon — CHANGES_REQUESTED** [review]
Minor stuff, but otherwise 💚

**Review by silvenon — DISMISSED** [review]

**Comment by vepor** [comment]
Action works properly, but surge doesn't want to actually publish anything. Will fix it later.

**Review by silvenon — APPROVED** [review]

**Pull request merged**